src/AccessLogs.sol

```diff
@@ -10,7 +10,8 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-/// @dev This file is generated from helper_scripts/generate_AccessLogs.sh, one should not modify the content directly
+//:#include macro.pp
+DEV_COMMENT(src/AccessLogs.sol)
```

Comment:
Do you think it's possible to put this line in a Solidity comment? I think the preprocessor will still see it and do substitutions, and the file will remain valid Solidity. What do you think?